### ISSUE TITLE
Slider Update

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#693]
**Issue:**
1. No focus control indicator so user does not know when they are focused on the slider.

2. SR only reads the number that the slider is currently on, not the min or max. There is also no indication that there is another slider (for the double sided slider variation.)

**Solution:**
1. Add focus control indicator to component

2. OQ: should the SR read both the min and max values? How do we indicate to the user that there are two sliders for the double sided slider variation?
